eox_core/middleware.py

```diff
@@ -36,13 +36,28 @@ class PathRedirectionMiddleware(object):
 
     def process_request(self, request):
         """
-        This middleware processes the path of every request and determines if there
-        is a configured action to take.
+        Process the path of every request and
+        determine the correct custom redirect.
         """
 
-        if not configuration_helper.has_override_value("EDNX_CUSTOM_PATH_REDIRECTS"):
-            return None
+        custom_configs = {
+            "MKTG_REDIRECTS": "process_mktg_redirect",
+            "EDNX_CUSTOM_PATH_REDIRECTS": "process_custom_path_redirect",
+        }
+
+        for key, value in six.iteritems(custom_configs):
```

Comment:
@jfavellar90 As far as I understand that should not happen, but in the eventual case, "MKTG_REDIRECTS" would have the priority, for no specific reason since each feature is equal for us, Do you agree?

Reply:
@anmrdz Ok I agree, but please add this note to the method docstring

Reply:
Once this is one, I approve

Reply:
@jfavellar90 Note added.